meta/meta.go

```diff
@@ -181,6 +186,26 @@ func (m *Meta) GetAutoTableID(dbID int64, tableID int64) (int64, error) {
 	return m.txn.HGetInt64(m.dbKey(dbID), m.autoTableIDKey(tableID))
 }
 
+// GenSequenceID adds step to the sequence value and returns the sum.
+func (m *Meta) GenSequenceID(dbID, sequenceID, step int64) (int64, error) {
+	// Check if DB exists.
+	dbKey := m.dbKey(dbID)
+	if err := m.checkDBExists(dbKey); err != nil {
+		return 0, errors.Trace(err)
+	}
+	// Check if sequence exists.
+	tableKey := m.tableKey(sequenceID)
+	if err := m.checkTableExists(dbKey, tableKey); err != nil {
+		return 0, errors.Trace(err)
+	}
+	return m.txn.HInc(dbKey, m.sequenceKey(sequenceID), step)
+}
+
+// GetSequenceID gets current sequence value with sequence id.
+func (m *Meta) GetSequenceID(dbID int64, sequenceID int64) (int64, error) {
```

Comment:
This function does the same work as `GetAutoTableID`. Could we remove it?

Reply:
it's a mistake, addressed.